package.json

```diff
@@ -41,7 +41,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "prepare": "husky install",
+    "setup-format-hook": "husky install",
```

Comment:
I think we could make this a [`postinstall` script](https://docs.npmjs.com/cli/v6/using-npm/scripts#npm-install), to ensure it gets set up for everyone. Not sure if this is the ideal way to solve this, but I think it should be somehow automated.

Reply:
Agreed. Initially I didn't want to force it on anybody but if we all agree on it, I'm fine with automating the install! Then we can just forget about it.